microsite/data/plugins/armo-kubescape

```diff
@@ -0,0 +1,12 @@
+title: Armo Kubescape Plugin
+author: Armo
+authorUrl: https://github.com/armosec
+category: Monitoring
+description: open-source Kubernetes Security platform, made for DevOps.
+  Our patented technology and open-source solutions fit natively within the CI/CD pipeline and existing development tools, assuring DevOps, DevSecOps, and developers that every Kubernetes cluster, container, and microservice is born and remains secure, from development to production and from configuration to run-time, every time
+documentation: https://github.com/armosec/backstage-plugin#readme
```

Comment:
Probably want to link here instead right? https://github.com/armosec/backstage-plugin/tree/master/plugins/kubescape-plugin